app/views/shared/_messages.html.slim

```diff
@@ -2,4 +2,4 @@
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
       div class="alert alert-#{name}" role='alert'
-        = msg
+        = sanitize(msg)
```

Comment:
`html_safe` when called inside `safe_join` is fine. Otherwise, Rubocop would complain. See the documentation: http://apidock.com/rails/v4.2.7/ActionView/Helpers/OutputSafetyHelper/safe_join

versus:
http://apidock.com/rails/ActionView/Helpers/SanitizeHelper/sanitize